### PR TITLE
Fix unhelpful visuals

### LINKED ISSUE #331: [PE-D][Tester B] Unhelpful visual in UG add command
In the UG under add command, the image provided does not show the complete result of executing the successful add command. It also does not show the newly added student `John Doe` displayed in the list of students.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: SQ77/ped#11

### PR DESCRIPTION
**Pull request by TY1Fan**
closes #331 

* Added before and after pics to make visuals more helpful. (for add, edit, pay, owe, settle)
* Standardized screenshots.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -146,8 +146,11 @@ Adds a student to the address book.
 **Example:**
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography r/100 paid/100 owed/0`
 
-**Output:**
-![addResult.jpeg](images/addResult.jpeg)
+**Before (using an empty UGTeach):**
+![addBefore.png](images/addBefore.png)
+
+**After:**
+![addBAfter.png](images/addAfter.png)
```
Comment (Feng1231) [inline docs/UserGuide.md:153]: Minor error here, addBAfter should be addAfter

**Review by Feng1231 — APPROVED** [review]
Thanks for the update, but there's a small error

File: docs/UserGuide.md (hunk 1 above)
Comment (TY1Fan) [reply docs/UserGuide.md]: Thanks for pointing this out! Ive fixed this

**Review by Soumyadip-cmd — COMMENTED** [review]
LGTM. Thank you for your hard work 🫡

**Review by Soumyadip-cmd — APPROVED** [review]

**Pull request merged**